astropy/nddata/tests/test_ccddata.py

```diff
@@ -881,3 +882,11 @@ def test_stddevuncertainty_compat_descriptor_no_weakref():
     uncert._parent_nddata = ccd
     assert uncert.parent_nddata is ccd
     uncert._parent_nddata = None
+
+
+def test_CCDataread_returns_image():
+    # Test if CCData.read returns a image when reading a fits file containing
+    #a table and image, in that order.
+    fits_table_image = get_pkg_data_filename('data/table_image.fits', package = 'astropy.nddata.tests')
```

Comment:
The `package=` part should be unnecessary. Hard to review why test failed until you have `rebase` and cleaned this up. Thanks!

Reply:
Thank you @pllim , I will do it right away!

Reply:
p.s. You also seem to be committing from 3 different email addresses, two of which are not associated with your GitHub account. If you want proper GitHub author credit for all the commits, I recommend that you add the other 2 emails to your GitHub account. However, this is entirely up to you and does not affect the review of this PR. FYI.

Reply:
You are right, I had to use another computer with a different git configuration. I will fix it!